pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -432,6 +456,11 @@ let
 
 in buildPythonPackage {
   inherit version pname;
+
+  # By default we expect this to be a no-op.
+  # For CUDA and Darwin builds this enforces a compatible toolchain.
+  inherit stdenv;
```

Comment:
Unintended change: Darwin used to only override buildBazelPackage, not buildPythonPackage. The new comment is probably misleading